src/allencell_ml_segmenter/_tests/prediction/test_model.py

```diff
@@ -56,100 +56,3 @@ def test_output_directory(prediction_model: PredictionModel) -> None:
 
     # ASSERT
     assert prediction_model.get_output_directory() == dummy_path
-
-
-def test_model_path(prediction_model: PredictionModel) -> None:
-    """
-    Tests that the model path is set and retrieved properly, and that the correct event is fired off.
-    """
-    # ARRANGE
-    event_under_test: Event = Event.ACTION_PREDICTION_MODEL_FILE
-    subscriber: FakeSubscriber = FakeSubscriber()
-    prediction_model.subscribe(event_under_test, subscriber, subscriber.handle)
-    dummy_path: Path = Path("example path")
-
-    # ACT
-    prediction_model.set_model_path(dummy_path)
-
-    # ASSERT
-    assert prediction_model.get_model_path() == dummy_path
-    assert subscriber.was_handled(event_under_test)
-
-
-def test_preprocessing_method(prediction_model: PredictionModel) -> None:
-    """
-    Tests that the preprocessing method is set and retrieved properly, and that the correct event is fired off.
-    """
-    # ARRANGE
-    event_under_test: Event = Event.ACTION_PREDICTION_PREPROCESSING_METHOD
-    subscriber: FakeSubscriber = FakeSubscriber()
-    prediction_model.subscribe(event_under_test, subscriber, subscriber.handle)
-
-    # ACT
-    prediction_model.set_preprocessing_method("example method")
-
-    # ASSERT
-    assert prediction_model.get_preprocessing_method() == "example method"
-    assert subscriber.was_handled(event_under_test)
-
-
-def test_postprocessing_method(prediction_model: PredictionModel) -> None:
-    """
-    Tests that the postprocessing method is set and retrieved properly, and that the correct event is fired off.
-    """
-    # ARRANGE
-    event_under_test: Event = Event.ACTION_PREDICTION_POSTPROCESSING_METHOD
-    subscriber: FakeSubscriber = FakeSubscriber()
-    prediction_model.subscribe(event_under_test, subscriber, subscriber.handle)
-
-    # ACT
-    prediction_model.set_postprocessing_method("example method")
-
-    # ASSERT
-    assert prediction_model.get_postprocessing_method() == "example method"
```

Comment:
All the deletes here were covering dead code?  Was that code deleted in this pr?

Reply:
this is all dead code which we wont use anymore